__tests__/app.test.js

```diff
@@ -0,0 +1,40 @@
+const request = require('supertest');
+const app = require('../app/app');
+
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data');
+
+const db = require('../db/connection');
+
+afterAll(() => db.end());
+
+beforeEach(() => seed(testData));
+
+describe('GET/api/topics', () => {
+  test('status 200, returns an array of object in correct format', () => {
+    return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then((result) => {
+        const topics = result.body;
+        expect(topics).toBeInstanceOf(Array);
```

Comment:
is this line necessary if we're iterating through successfully in the next line?